en/docs/concepts/user-management/roles-and-permissions.md

```diff
@@ -0,0 +1,32 @@
+# Roles and Permissions
+
```

Comment:
shall we start the doc by simple definitions of what is a role and what is a permission? WDYT?